src/test/java/org/hillel/homework/mapper/OrderItemMapperTest.java

```diff
@@ -0,0 +1,62 @@
+package org.hillel.homework.mapper;
+
+import org.hillel.homework.dto.request.OrderItemRequest;
+import org.hillel.homework.dto.response.OrderItemResponse;
+import org.hillel.homework.entity.OrderItem;
+import org.hillel.homework.entity.Product;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class OrderItemMapperTest {
+
+    private OrderItemMapper orderItemMapper;
+
+    @BeforeEach
+    public void setUp() {
+        orderItemMapper = new OrderItemMapper();
+    }
+
+    @Test
+    public void requestToEntity_validObject_entityReturned() {
+        OrderItemRequest request = OrderItemRequest.builder()
+                .productId(1L)
+                .quantity(new BigDecimal("2.5"))
+                .build();
+
+        OrderItem entity = orderItemMapper.requestToEntity(request);
+
+        assertEquals(1L, entity.getProduct().getId());
```

Comment:
Consider using assertThat(result).isEqualto(expected); it provides more readable approach and better failure handling